pkg/controller/storagecluster/reconcile.go

```diff
@@ -51,6 +56,8 @@ var validTopologyLabelKeys = []string{
 	"topology.rook.io",
 }
 
+var throttleDiskTypes = []string{"gp2", "io1"}
```

Comment:
I can't say that I like hard coding two device types of one provisioner for throttling. This is not very robust, and not expandable without code changes... Can we have a configuration input (with some defaults) that specifies provisioners and disk types that we want to be throttled?

Reply:
As per the discussion on BZ, it seemed like throttling these two were good enough. I could add a `slowDeviceClasses` config that could used to determine which devices to throttle in combination with `tuneSlowDeviceClass`. But, is it really required/worth?

@jarrpa @leseb Thoughts?